client/my-sites/checklist/index.jsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { filter, noop, range } from 'lodash';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import ChecklistHeader from './header';
+import ChecklistTask from './task';
+import ChecklistPlaceholder from './placeholder';
+
+export class Checklist extends Component {
+	static propTypes = {
+		tasks: PropTypes.array,
+		onClickTask: PropTypes.func,
```

Comment:
Most folks would expect a name of `onClick`, unless we had multiple click handlers to pass through.